src/modules/cpu/kernel/noise_gaussian.hpp

```diff
@@ -0,0 +1,1442 @@
+#include "rppdefs.h"
+#include "rpp_cpu_simd.hpp"
+#include "rpp_cpu_common.hpp"
+
+RppStatus gaussian_noise_u8_u8_host_tensor(Rpp8u *srcPtr,
```

Comment:
too huge file with similar code repeated in many functions. We need better ways to condense it.

Reply:
Same as prev. Similar lines also reduced in salt_and_pepper noise.